### ISSUE TITLE
How to make the sidebar open by default?

### ISSUE THREAD

**Issue by mgd722** [vikdiesel/admin-one-vue-bulma-dashboard#6]
We've messed around with this a bit, but can't seem to come to a good solution. Currently we just execute the toggle function when the app opens, but on slower computers this results in you seeing the sidebar animate open. Is there a simple config parameter we've missed somewhere that would allow us to set the sidebar to be open by default?

Thanks Viktor! We've been loving the dashboard thus far-- you did a great job! Please let me know if you have a preferred channel for requests like this (other than GitHub issues).

**Comment by vikdiesel** [comment]
Hi Mike, Thank you )

Am I right, you would like to have aside menu in expanded state by default?

Support channels: GitHub issues for free items, e-mail for paid ones

**Comment by mgd722** [comment]
That's correct. I've played around with it a bit but can't seem to find the right combination of settings to make the sidebar open by default. If this isn't supported, no worries-- just figured if anyone would know of the best way to do this it'd be you!

**Comment by lhoucinecherif** [comment]
Hi mgd722,

for anyone having issue with the side bar, you should play with some classes (_has-aside-left has-aside-mobile-transition has-navbar-fixed-top has-aside-expanded_) at the <html> tag, so you  can show it properly 
if you are using a javascript framework like vuejs you can add those classes on the index.html file manually or use this snippet on create event for example :
`let htmlTag = document.getElementsByTagName("html")[0];
            let htmlClasses = ['has-aside-left', 'has-aside-mobile-transition', 'has-navbar-fixed-top', 'has-aside-expanded'];
            for (let htmlClass of  htmlClasses){
                if (!htmlTag.classList.contains(htmlClass)){
                    htmlTag.classList.add(htmlClass)
                }
            }`
